examples/seventh/util/util.go

```diff
@@ -0,0 +1,41 @@
+package seventh
```

Comment:
no need for a extra file and package. we should add it in the main file since it is just and example and the util style packages are not good package names.

Reply:
yes, i agree. the problem is main is not importable. the idea for this file is that other example services can use this util to interact with the service. I thought it would be better to make building of examples easier and more fluent.

Reply:
maybe you have some idea on the package naming instead of util ?

Reply:
first of all, you could use one of the existing services, example 1 or 2, to add caching as an example since both of them have HTTP.
The function here should live in the service that calling so in example 1.